src/compiler/core.ts

```diff
@@ -633,6 +633,10 @@ module ts {
         }
     }
 
+    export function getDefaultLibraryFilename(options: CompilerOptions): string {
+        return options.target === ScriptTarget.ES6 ? "lib.es6.d.ts" : "lib.d.ts";
+    }
+

```

Comment:
should we be more generic here for future targets? something that converts ScriptTaget into string and use that with lib.scriptTagetString.d.ts ?